### PR TITLE
🏫  add license info & other key documents

### PR DESCRIPTION
**Pull request by stevejpurves**
Aligning `thebe-core` with `thebe` license

### CODE REVIEW

**Comment by stevejpurves** [comment]
@choldgraf I've moved the license over from `thebe` as I think the same would apply here

What other documents are we missing from this an an EB repository?

 - The original thebe has a code of conduct document https://github.com/executablebooks/thebe/blob/master/CODE_OF_CONDUCT.md which should at least be updated with a new contact, but I don't see this as a feature on other EB repos? potentially related to https://github.com/executablebooks/team-compass/issues/2?
 
- I'm not adding a CHANGELOG until we get to a proper release, which will be when `thebe` starts to consume this.

**Comment by stevejpurves** [comment]
Also @choldgraf is there an appropriate way to update the Acknowledgments section to reflect the latest round of work, indicating involvement and funding source?

File: README.md [hunk 1]
```diff
@@ -29,3 +29,11 @@ To learn how to build `thebe-core` itself and contribute to the development of t
...
 
 For more information on how to invoke thebe and connect to a kernel see [demo/index.html](./demo/index.html) and [demo/demo.js](./demo/demo.js).
+
+# License
+
+`thebe-core` is licensed under BSD 3 Clause Revised License - see [LICENSE](./LICENSE).
+
+# Acknowledgements
+
+`thebe-core` is an out of core refactor of the `thebe` library, containing refactored code from the original. `thebe` was developed as a part of OpenDreamKit – Horizon 2020 European Research Infrastructure project (676541). It is currently stewarded by the Executable Books Project. Additional support was provided by the U.S. Department of Education Open Textbooks Pilot Program funding for the LibreTexts project (P116T180029).
```
Comment (choldgraf) [inline README.md]:
```suggestion
`thebe-core` is an out of core refactor of the `thebe` library, containing refactored code from the original.

It is currently stewarded by [the Executable Books Team](https://executablebooks.org/en/latest/team.html).

**Previous funding**

`thebe` was initially developed as a part of [OpenDreamKit](https://opendreamkit.org/) – Horizon 2020 European Research Infrastructure project (676541).

Additional support was provided by the U.S. Department of Education Open Textbooks Pilot Program funding for the LibreTexts project (P116T180029).
```

**Review by choldgraf — APPROVED** [review]
Some quick thoughts on the acknowledgements section below.

For the Code of Conduct, how about we point it to the page in executablebooks.org that is being added by the PR here:

- https://github.com/executablebooks/meta/pull/770/files

I also opened up another issue to track attribution of organizations in our Team page. Would love thoughts there:

- https://github.com/executablebooks/meta/issues/774

**Pull request merged**